cfme/tests/cli/test_appliance_cli.py

```diff
@@ -199,3 +202,35 @@ def is_failover_started(appliance):
     wait_for(is_failover_started, func_args=[apps[2]], timeout=450, handle_exception=True)
     apps[2].wait_for_evm_service()
     apps[2].wait_for_web_ui()
+
+
+def test_appliance_console_cli_configure_ipa(configured_appliance):
+    auth_provider_conf = auth_data.auth_providers.freeipa01
```

Comment:
This is referencing a specific key within auth_providers yaml, and is not appropriate - it won't be available to upstream community and needs to be wrapped in a fixture that finds an appropriate server for test.